### ISSUE TITLE
Get version number

### ISSUE THREAD

**Issue by rolegic** [gnab/remark#200]
Is it possible to add version number to ease traceability/identification?

In js header (traceability):
```javascript
/*! remark 0.12 | (c) 2011-2015 Ole Petter Bang */
```

In js code (identification, e.g. display remark version):
```javascript
  version : "0.12"
```

**Comment by gnab** [comment]
This is fixed in #225 and will be included in the upcoming release 0.11.

**Comment by gnab** [comment]
Version 0.11 which fixed this has now been released.
